src/status_im/contexts/keycard/sheets/migrate/view.cljs

```diff
@@ -0,0 +1,33 @@
+(ns status-im.contexts.keycard.sheets.migrate.view
+  (:require [quo.core :as quo]
+            [react-native.core :as rn]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn view
+  []
+  (let [profile-name        (rf/sub [:profile/name])
+        profile-picture     (rf/sub [:profile/image])
+        customization-color (rf/sub [:profile/customization-color])]
+    [:<>
+     [quo/drawer-top
+      {:type                :context-tag
+       :context-tag-type    :default
+       :title               (i18n/label :t/migrate-key-pair-keycard)
+       :full-name           profile-name
+       :profile-picture     profile-picture
+       :customization-color customization-color}]
+     [rn/view {:style {:padding-horizontal 20}}
+      [quo/text {}
+       (i18n/label :t/migrate-key-pair-keycard-default-key {:name profile-name})]
+      [quo/information-box
+       {:type  :default
+        :style {:margin-top 20 :margin-bottom 12}}
+       (i18n/label :t/migrate-key-pair-keycard-info)]]
+     [quo/bottom-actions
+      {:actions          :two-actions
+       :button-one-label (i18n/label :t/continue)
+       :button-one-props {:on-press #()}
```

Comment:
What should happen when the continue is pressed? Is it okay to leave it as a no-op for now?

Reply:
yes, we are trying to break down a big task and this smaller task ends here, the rest will be in the next ones. that's why this feature under the flag